lib/resolver.js

```diff
@@ -0,0 +1,25 @@
+// Load modules.
+var NoOpenIDError = require('./errors/no_openid_error');
+
+
+/**
+ * Discover OpenID Connect provider configuration from Authentiq Provider.
```

Comment:
@skion this is a custom resolver as we haven't implemented WebFinger spec yet. You can check [here](https://github.com/jaredhanson/passport-openidconnect/blob/master/lib/resolver.js#L10-L27) for more details